crates/sim2h/src/lib.rs

```diff
@@ -34,22 +37,37 @@ use parking_lot::RwLock;
 use rand::Rng;
 use std::{collections::HashMap, convert::TryFrom};
 
+const RECALC_RRDHT_ARC_RADIUS_INTERVAL_MS: u64 = 20000; // 20 seconds
+
 pub struct Sim2h {
+    crypto: Box<dyn CryptoSystem>,
     pub bound_uri: Option<Lib3hUri>,
     connection_states: RwLock<HashMap<Lib3hUri, ConnectionState>>,
     spaces: HashMap<SpaceHash, RwLock<Space>>,
     stream_manager: StreamManager<std::net::TcpStream>,
-    num_ticks: u32,
+    num_ticks: u64,
+    /// sim2h currently uses the same radius for all connections
+    rrdht_arc_radius: u32,
```

Comment:
Hm, I think this is the wrong place for this variable to be. The sim2h server can server many spaces, and each space will have a different aggregated arc radius, no?